lib/transport/reusable_data/reusable_data.ex

```diff
@@ -113,6 +120,56 @@ defmodule Transport.ReusableData do
     end
   end
 
+  @doc """
+  Builds a licence.
+
+  ## Examples
+
+      iex> %{name: "Libertarian"}
+      ...> |> ReusableData.build_licence
+      ...> |> Map.get(:name)
+      "Libertarian"
+
+      iex> %{name: "Libertarian", random_key: "asdfasdf"}
+      ...> |> ReusableData.build_licence
+      ...> |> Map.get(:random_key)
+      nil
+
+  """
+  @spec build_licence(map()) :: %Licence{}
+  def build_licence(%{} = attrs) do
```

Comment:
it can fail, it should be `build_licence!`

Reply:
Not anymore.